### ISSUE TITLE
Profile missing description terms

### ISSUE THREAD

**Issue by iadawn** [w3c/w3c-website#248]
On the profile page there is a `<dl>` which presents information about the user; affilliation, location, github id etc. There is an icon to visually indicate what each definition row contains. There is no alternative text for this icon so it may not be clear what each definition is.

<img width="345" alt="screen shot of the profile definition list" src="https://user-images.githubusercontent.com/2444840/222102233-3026af33-b040-4e2f-9d3b-fd3d5a0db853.png">

**Expected behavior**

Include alternative text for the icons. The approach taken for the preferred email address icon should be sufficient.

```
<span class="visuallyhidden">preferred email address</span>
<span class="fas fa-star" aria-hidden="true" title="preferred email"></span>
```

**Comment by deniak** [comment]
@iadawn thanks for the suggestions. The text has been added.
